src/libcoretest/any.rs

```diff
@@ -18,7 +18,7 @@ static TEST: &'static str = "Test";
 
 #[test]
 fn any_referenced() {
-    let (a, b, c) = (&5u as &Any, &TEST as &Any, &Test as &Any);
+    let (a, b, c) = (&5us as &Any, &TEST as &Any, &Test as &Any);
```

Comment:
This should probably be changed to use `i32`, there's no need for a platform dependent integer type here.